Content.Server/GameObjects/Components/Power/PowerNetComponents/BatteryDischargerComponent.cs

```diff
@@ -31,14 +33,16 @@ public override void ExposeData(ObjectSerializer serializer)
         public override void Initialize()
         {
             base.Initialize();
-
-            _battery = Owner.EnsureComponent<BatteryComponent>();
-            _supplier = Owner.EnsureComponent<PowerSupplierComponent>();
+            _battery = Owner.EnsureComponentWarn<BatteryComponent>();
+            _supplier = Owner.EnsureComponentWarn<PowerSupplierComponent>();
```

Comment:
nitpick: you are assigning this twice because compdeps do it in the background. call the function but dont use the return value imo